webapp/routes.go

```diff
@@ -40,7 +40,8 @@ func RegisterRoutes() {
 	shared.AddRoute("/interop/{path:.*}", "interop", interopHandler)
 
 	// List of all test runs, by SHA[0:10]
-	shared.AddRoute("/test-runs", "test-runs", testRunsHandler)
+	shared.AddRoute("/runs", "test-runs", testRunsHandler)
```

Comment:
Thanks :) Can you point to https://github.com/web-platform-tests/wpt.fyi/issues/670 in the commit message?

Reply:
Done.